ocs_ci/ocs/platform_nodes.py

```diff
@@ -1660,6 +1655,24 @@ def read_default_config(self, default_config_path):
 
         return default_config_dict
 
+    def create_and_attach_nodes_to_cluster(self, node_conf, node_type, num_nodes):
```

Comment:
since this has the same implementation as in the `VMWareNodes`, I suggest moving it to the `NodesBase`

Reply:
This is present in NodeBase class